app/Enums/Prefectures.php

```diff
@@ -0,0 +1,109 @@
+<?php declare(strict_types=1);
+
+namespace App\Enums;
+
+use BenSampo\Enum\Enum;
+
+final class Prefectures extends Enum
+{
+    const HOKKAIDO = 1;
+    const AOMORI = 2;
+    const IWATE = 3;
+    const MIYAGI = 4;
+    const AKITA = 5;
+    const YAMAGATA = 6;
+    const FUKUSHIMA = 7;
+    const IBARAKI = 8;
+    const TOCHIGI = 9;
+    const GUNMA = 10;
+    const SAITAMA = 11;
+    const CHIBA = 12;
+    const TOKYO = 13;
+    const KANAGAWA = 14;
+    const NIGATA = 15;
```

Comment:
```suggestion
    const NIIGATA = 15;
```
新潟県のホームページではこの表記なので一応合わせておきましょうか。